src/main/resources/application.yaml

```
@@ -1,33 +1,68 @@
-server:
-  port: 8080
 spring:
-  servlet:
-    multipart:
-      enabled: true
-      max-file-size: 10MB
-      max-request-size: 20MB
-  web:
-    resources:
-      cache:
-        period: 0
-  datasource:
-    url: jdbc:postgresql://localhost:5432/discodeit
-    username: discodeit_user
-    password: discodeit1234
-    driver-class-name: org.postgresql.Driver
+  application:
+    name: discodeit
+
+  profiles:
+    active: test
+
   jpa:
-    database-platform: org.hibernate.dialect.PostgreSQLDialect
-    hibernate:
-      ddl-auto: update
     show-sql: true
-logging:
-  level:
-    org.hibernate.SQL: DEBUG # org : hibernate: sqL: 했을 떈 오류 떴는데 이렇게 하니까 WARM안뜸
+
+  servlet:
+    multipart:
+      max-file-size: 10MB # 파일 하나의 최대 크기
+      max-request-size: 20MB # 한 벙네 최대 업로드 가능 용량
+
 discodeit:
   storage:
     type: local
     local:
-      root-path: "C:/Users/User/OneDrive/Desktop/codeit/1-sprint-mission/src/main/resources/static"
+      root-path: /storage
+
+logging:
+  level:
+    root: INFO
+
+
+management:
+  endpoints:
+    web:
+      exposure:
+        include:
+          - "health"
+          - "info"
+          - "metrics"
+          - "loggers"
+  endpoint:
+    health:
+      show-details: "ALWAYS"
+  info:
+    git:
+      mode: full # Git 정보를 가져올 수 있도록 설정
+
+info:
+  app:
+    name: Discodeit
+    version: 1.7.0
+    java-version: 17
+    spring-boot-version: 3.4.0
+
+  datasource:
+    url: jdbc:h2:mem:devDB;MODE=PostgreSQL
+    driver-class-name: org.h2.Driver
```

yaml 파일 내에서 다른 프로퍼티를 가져올 수 있다는 걸 알고 계신가요?

```suggestion
    url: ${spring.datasource.url}
    driver-class-name: ${spring.datasource.driver-class-name}
```